src/FINDOLOGIC/Export/Helpers/DataHelper.php

```diff
@@ -99,4 +107,17 @@ public static function checkForIllegalCsvPropertyKeys($propertyKey)
             throw new BadPropertyKeyException($propertyKey);
         }
     }
+
+    /**
+     * Verifies that the provided value is not a object.
+     *
+     * @param mixed $value The value to check.
+     * @throw ObjectNotAllowedException In case the provided value is a object.
+     */
+    public static function checkForObjectType($value)
+    {
+        if (is_object($value)) {
```

Comment:
Just for keeping the convention:

In this method you could return `is_object($value)` and add it as condition for `UsergroupAwareSimpleValue::validate`, as already is done in `DataHelper::checkForEmptyValue`.

E.g.

```php
return DataHelper::checkForEmptyValue($value) && DataHelper::checkForObjectType($value);
```

Reply:
That might be a good idea.

Also, it would be better if the method checked if the value was an acceptable one (`is_string()` and `is_numeric()`) instead of checking if it was one of, possible multiple, disallowed types.

Another thing is: What if the passed object has a useful `::__toString()` method that actually returns something that is worth being exported? If there was a nice way to check this, the library's convenience could be improved, but I guess it's hard to judge if a `::__toString()` method makes sense at runtime. So we can probably expect users to explicitly call it.